### PR TITLE
[Portal] Render single div for container element

### PR DESCRIPTION
**Pull request by dleroux**
### WHY are these changes introduced?

Fixes #0000 

## 

### How to 🎩

🖥 [Local development instructions](https://github.com/Shopify/polaris-react/blob/master/README.md#development)
🗒 [General tophatting guidelines](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting.md)
📄 [Changelog guidelines](https://github.com/Shopify/polaris-react/blob/master/documentation/Versioning%20and%20changelog.md)

<details>
<summary>Copy-paste this code in <code>playground/Playground.tsx</code>:</summary>

```jsx
import React from 'react';
import {Page} from '../src';

export function Playground() {
  return (
    <Page title="Playground">
      {/* Add the code you want to test in here */}
    </Page>
  );
}

```

</details>

### 🎩 checklist

* [ ] Tested on [mobile](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting.md#cross-browser-testing)
* [ ] Tested on [multiple browsers](https://help.shopify.com/en/manual/intro-to-shopify/shopify-admin/supported-browsers)
* [ ] Tested for [accessibility](https://github.com/Shopify/polaris-react/blob/master/documentation/Accessibility%20testing.md)
* [ ] Updated the component's `README.md` with documentation changes
* [ ] [Tophatted documentation](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting%20documentation.md) changes in the style guide
* [ ] For visual design changes, pinged one of @ HYPD, @ mirualves, @ sarahill, or @ ry5n to update the Polaris UI kit

### CODE REVIEW

File: src/components/Portal/Portal.tsx [hunk 1]
```diff
@@ -26,37 +28,53 @@ export class Portal extends PureComponent<PortalProps, State> {
...
 
+  private portalContainerNode: HTMLElement | null = null;
+
   private portalId =
     this.props.idPrefix !== ''
       ? `${this.props.idPrefix}-${getUniqueID()}`
       : getUniqueID();
 
   componentDidMount() {
+    const constainerNode = document.getElementById(UNIQUE_CONTAINER_ID);
```
Comment (qq99) [inline src/components/Portal/Portal.tsx:39]:
```suggestion
    const containerNode = document.getElementById(UNIQUE_CONTAINER_ID);
```

Comment (dleroux) [reply src/components/Portal/Portal.tsx:39]: 🙈

File: UNRELEASED.md [hunk 2]
```diff
@@ -8,7 +8,8 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 ### Enhancements
 
-Updated `Textfield` with a `type` of `number` to not render a spinner if step is set to `0` ([#3477](https://github.com/Shopify/polaris-react/pull/3477))
+- Changed `Portal` to render all `Portals` in a single div ([#3465](https://github.com/Shopify/polaris-react/pull/3465))
```
Comment (alex-page) [inline UNRELEASED.md:11]: Do we want to add the conversion to functional component

**Review by alex-page — APPROVED** [review]

**Review by kyledurand — APPROVED** [review]
👏

File: UNRELEASED.md (hunk 2 above)
Comment (dleroux) [inline UNRELEASED.md:11]:
```suggestion
- Converted `Portal` to a functional component ([#3465](https://github.com/Shopify/polaris-react/pull/3465))
- Changed `Portal` to render all `Portals` in a single div ([#3465](https://github.com/Shopify/polaris-react/pull/3465))
```

Comment (dleroux) [reply UNRELEASED.md:12]: 👍

**Comment by alex-page** [comment]
Closing this for https://github.com/Shopify/polaris-react/pull/3544

**Pull request closed without merging**